mock-web-server-rule/build.gradle.kts

```diff
@@ -0,0 +1,48 @@
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
+plugins {
+    alias(libs.plugins.android.library)
```

Comment:
question: is it possible, to make this library Java library instead of android library? This will allow using this rule in non android projects.